netebpfext/user/fwp_um.h

```diff
@@ -77,6 +77,15 @@ typedef class _fwp_engine
     FWP_ACTION_TYPE
     classify_test_packet(_In_ const GUID* layer_guid, NET_IFINDEX if_index);
 
+    void
```

Comment:
This series of ```test_xxx``` functions do not return a value.  So I'm assuming the test 'pass/fail' status would be indicated via exceptions (given the explicit 'user mode' nature of this component)?

Reply:
Correct.   I'm writing that part now as part of a separate PR.  The prototypes might change if I find I need them to.